lib/beaker/options/parser.rb

```diff
@@ -181,9 +203,12 @@ def initialize
       # @raise [ArgumentError] Raises error on bad input
       def parse_args(args = ARGV)
         @options                        = @presets.presets
+        tag_sources(@presets.presets, "default")
```

Comment:
Using the value "default" creates an unnecessary mapping here between the presets that already exist and this keyword. We should eliminate unnecessary mappings.